### DISCUSSION TITLE
[v9] I want to compile all my test cases before run all of them

### DISCUSSION THREAD

**Discussion by lararojasmr** [webdriverio/webdriverio#13614] [Q&A]
I'm having several problem running wdio without Bidi because esbuild bug is affecting the execute commands, so i'm trying to solve compiling all with tcs and after that running the wdio command from compiled files but when i try to do that an unexpected error is displayed. Any help on this: 

Uncaught Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'tsx' imported from

I saw a environment variable is set.

**Comment by christian-bromann** [comment]
@lararojasmr mind providing a minimal reproducible example?

**Comment by FluffyJay1** [comment]
Are you by any chance running your tests from a working directory outside of your project? I know `wdio-local-runner` will spawn child node processes that 1) inherit the parent process' working dir and 2) run with `--import tsx` (since wdio v9). Thus `tsx` needs to be resolvable from your working directory, e.g. the root of your node project.
